base/show.jl

```diff
@@ -902,25 +902,15 @@ unquoted(ex::Expr)       = ex.args[1]
 
 ## AST printing helpers ##
 
-typeemphasize(io::IO) = get(io, :TYPEEMPHASIZE, false) === true
-
 const indent_width = 4
 
-function show_expr_type(io::IO, @nospecialize(ty), emph::Bool)
+function show_expr_type(io::IO, @nospecialize(ty))
     if ty === Function
         print(io, "::F")
     elseif ty === Core.IntrinsicFunction
         print(io, "::I")
     else
```

Comment:
Can we just drop this function entirely now, and call `print(io, "::", ty)` directly? It's weird to have this one place in the code that prints two random types supertypes oddly.